pkg/deploy/saveversion.go

```diff
@@ -4,25 +4,27 @@ package deploy
 // Licensed under the Apache License 2.0.
 
 import (
-	"bytes"
 	"context"
-	"net/url"
+	"fmt"
 	"time"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/service"
 	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2021-09-01/storage"
-	azstorage "github.com/Azure/azure-sdk-for-go/storage"
 	"github.com/Azure/go-autorest/autorest/date"
+
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/azuresdk/azblob"
+	"github.com/Azure/ARO-RP/pkg/util/pointerutils"
```

Comment:
nit: this package is duplicated.  We now have `k8s.io/utils/ptr` which performs the exact same functionality.

Reply:
We had an internal discussion on this a while back (pinging you in the thread) - between K8S's, Autorest's, and our own, we decided to stick to ours since its implementation is literally just `func ToPtr[T any](t T) *T { return &t }` and doesn't tightly couple us to any 3rd party deps for the functionality.